backend/cognito/cognito.template.yml

```diff
@@ -22,6 +22,12 @@ Parameters:
     Type: String
     Default: ""
     Description: The ARN of the permissions boundary to apply when creating IAM roles [Secure Pipelines]
+  NotifyApiKey:
+    Type: String
+    Default: please-provide-notify-api-key
```

Comment:
```suggestion
```
This must not be a param, as its value is secret and it's resolved using a dynamic reference.

Reply:
addressed